lib/administrate/field/rich_text.rb

```diff
@@ -0,0 +1,21 @@
+require "administrate/field/base"
+
+module Administrate
+  module Field
+    class RichText < Administrate::Field::Base
+      def to_s
+        data&.to_plain_text
+      end
+
+      def truncate
+        to_s[0...truncation_length]
```

Comment:
Is there an advantage to using `#[]` access over [String#truncate](https://edgeapi.rubyonrails.org/classes/String.html#method-i-truncate)?

```suggestion
        to_s.truncate(truncation_length)
```